src/core-components/Button/Button.less

```diff
@@ -0,0 +1,27 @@
+.ant-btn {
+    border-radius: 10px !important;
+}
+.ant-btn.ant-btn-loading::before {
+    opacity: 0 !important;
+}
+.ant-btn.ant-btn-primary.ant-btn-loading:not([disabled]):not(.ant-btn-background-ghost) {
+    background: #FF5135;
```

Comment:
Let's add global variables.less file where we will define colors and other less vars